debian/libmiral3.symbols

```diff
@@ -397,6 +397,8 @@ libmiral.so.3 libmiral3 #MINVER#
  (c++)"miral::WindowInfo::attached_edges(MirPlacementGravity)@MIRAL_2.7" 2.7.0
  (c++)"miral::WindowInfo::exclusive_rect() const@MIRAL_2.7" 2.7.0
  (c++)"miral::WindowInfo::exclusive_rect(mir::optional_value<mir::geometry::Rectangle> const&)@MIRAL_2.7" 2.7.0
+ (c++)"miral::WindowSpecification::application_id[abi:cxx11]() const@MIRAL_2.7" 2.7.0
+ (c++)"miral::WindowSpecification::application_id[abi:cxx11]()@MIRAL_2.7" 2.7.0
```

Comment:
These ought to be in the 2.8.0 namespace. (It looks as though they are correct in symbols.map, so what's going wrong?)

Reply:
I had link time optimization off, so the script wasn't generating this file